### PR TITLE
Mixer Diff: 2023-10-30

### PR DESCRIPTION
**Pull request by pradh**

### CODE REVIEW

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/alb.json [hunk 1]
```diff
@@ -130,8 +130,6 @@
         "AmountInterestRepayment_Debt_OfficialCreditor_PubliclyGuaranteed_Multilateral_LongTermExternalDebt_LenderWorldBankIDA",
         "AmountInterestRepayment_Debt_PrivateCreditor_PrivateNonGuaranteed_Bond_LongTermExternalDebt",
         "AmountInterestRepayment_Debt_PrivateCreditor_PrivateNonGuaranteed_Bond_LongTermExternalDebt_LenderBondholders",
-        "AmountInterestRepayment_Debt_PrivateCreditor_PrivateNonGuaranteed_OneOrgCommercialBankAndOther_LongTermExternalDebt",
```
Comment (pradh) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/alb.json:133]: Is this expected @ajaits ?

File: internal/server/v2/facet/golden/series_facet/series_facet.json [hunk 2]
```diff
@@ -33,58 +33,66 @@
               ]
             },
             {
-              "facet_id": "217147238",
+              "facet_id": "1151455814",
               "observations": [
                 {
-                  "value": 5
+                  "value": 28
```
Comment (pradh) [inline internal/server/v2/facet/golden/series_facet/series_facet.json:39]: Question for @chejennifer from @shifucun on this: https://github.com/datacommonsorg/mixer/pull/1286#discussion_r1365860210

Comment (shifucun) [reply internal/server/v2/facet/golden/series_facet/series_facet.json:39]: This is good. @chejennifer has a plan to update the meaning of this value later.

**Review by shifucun — APPROVED** [review]

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/alb.json (hunk 1 above)
Comment (ajaits) [reply internal/server/v0/placestatvar/golden/get_place_stat_vars/alb.json:133]: yes  'OneOrg' was renamed to 'WorldBank' as per OneOrg's request. The SV was thus renamed to:
AmountInterestRepayment_Debt_PrivateCreditor_PrivateNonGuaranteed_WorldBankCommercialBankAndOther_LongTermExternalDebt

Both existed in the last release to allow OneOrg to switch, With infrequent rebuilt, the earlier SV is now deleted.

**Review by ajaits — APPROVED** [review]

**Pull request merged**